pkg/controller/codestatus/show.go

```diff
@@ -18,18 +18,37 @@ package codestatus
 
 import (
 	"net/http"
+	"time"
 
 	"github.com/google/exposure-notifications-verification-server/pkg/controller"
 )
 
-func (c *Controller) Show() http.Handler {
+func (c *Controller) HandleShow() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
 
+		uuids := r.URL.Query()["uuid"]
+		var uuid string
+		if len(uuids) == 1 {
+			uuid = uuids[0]
+		}
+
+		// TODO: err
+		code, _ := c.db.FindVerificationCodeByUUID(uuid)
+
 		m := controller.TemplateMapFromContext(ctx)
-		// TODO(whaught): we're going to load a list of recent codes to show
-		// and mostly replace this experience
 
-		c.h.RenderHTML(w, "cstatus", m)
+		// TODO(whaught): errors?
+		// TODO(whaught): logic
+		m["UUID"] = uuid
```

Comment:
Nit: we've generally been using Go variable-naming styles for templates, so:

- `uuid`
- `status`
- `expires`